vms/avm/states/state.go

```diff
@@ -119,6 +122,8 @@ type State interface {
 	// TODO: remove after v1.11.x is activated
 	Prune(lock sync.Locker, log logging.Logger) error
 
+	Checksums() (ids.ID, ids.ID)
```

Comment:
nit: it would be useful to add comment for Checksums() function here for future reference

Reply:
Added comment - also moved the checksum update to `AddTx` to keep both checksums updated pre-commit as the state is modified (rather than updating the utxo checksum pre commit and the tx checksum post commit).